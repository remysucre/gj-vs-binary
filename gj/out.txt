running query 1a 
Query: 1a
Loading title
Loading movie_info_idx
Loading it
Loading mc
Loading ct
building flat table on mc
building mc takes 0.000006583s
building trie on t
building t takes 0.6707412s
building trie on mi_idx
building mi_idx takes 0.13487732s
building trie on it
building it takes 0.000002791s
building trie on ct
building ct takes 0.000046333s
join takes 13.878833ms
total takes 0.81962144
output [[Str("(A Warner Bros.-First National Picture) (presents)")], [Str("A Clockwork Orange"), Num(1934)]]
running query 1b 
Query: 1b
Loading movie_info_idx
Loading it
Loading mc
Loading ct
Loading t
building flat table on mc
building mc takes 0.000016708s
building trie on t
building t takes 0.17370084s
building trie on mi_idx
building mi_idx takes 0.13074438s
building trie on it
building it takes 0.000003416s
building trie on ct
building ct takes 0.000003166s
join takes 393.156041ms
total takes 0.69769675
output [[Str("(Set Decoration Rentals) (uncredited)")], [Str("Disaster Movie"), Num(2008)]]
running query 1c 
Query: 1c
Loading movie_info_idx
Loading t
Loading mc
Loading ct
Loading it
building flat table on mc
building mc takes 0.000003708s
building trie on t
building t takes 0.077353455s
building trie on mi_idx
building mi_idx takes 0.13527325s
building trie on it
building it takes 0.00000325s
building trie on ct
building ct takes 0.000003s
join takes 5.149333ms
total takes 0.21783501
output [[Str("(co-production)")], [Num(2011), Str("Intouchables")]]
running query 1d 
Query: 1d
Loading t
Loading mc
Loading ct
Loading movie_info_idx
Loading it
building flat table on mc
building mc takes 0.000004875s
building trie on t
building t takes 0.3646993s
building trie on mi_idx
building mi_idx takes 0.13882984s
building trie on it
building it takes 0.000003625s
building trie on ct
building ct takes 0.000003375s
join takes 439.83925ms
total takes 0.94347095
output [[Str("(Set Decoration Rentals) (uncredited)")], [Num(2004), Str("Disaster Movie")]]
running query 2a 
Query: 2a
Loading movie_keyword
Loading movie_companies
Loading title
Loading cn
Loading k
building flat table on mk
building mk takes 0.000003916s
building trie on t
building t takes 0.6989261s
building trie on mc
building mc takes 0.4133517s
building trie on k
building k takes 0.000003333s
building trie on cn
building cn takes 0.000684083s
join takes 1.142809041s
total takes 2.255872
output [[Str("'Doc'")]]
running query 2b 
Query: 2b
Loading movie_keyword
Loading movie_companies
Loading title
Loading k
Loading cn
building flat table on mk
building mk takes 0.000005208s
building trie on t
building t takes 0.72419274s
building trie on mc
building mc takes 0.470457s
building trie on k
building k takes 0.000003958s
building trie on cn
building cn takes 0.000269208s
join takes 1.109356541s
total takes 2.3044028
output [[Str("'Doc'")]]
running query 2c 
Query: 2c
Loading movie_keyword
Loading title
Loading k
Loading movie_companies
Loading cn
building flat table on mk
building mk takes 0.000009625s
building trie on t
building t takes 0.6672597s
building trie on mc
building mc takes 0.46997s
building trie on k
building k takes 0.000010041s
building trie on cn
building cn takes 0.000004666s
join takes 1.097071916s
total takes 2.2344332
output []
running query 2d 
Query: 2d
Loading movie_keyword
Loading movie_companies
Loading cn
Loading title
Loading k
building flat table on mk
building mk takes 0.000007s
building trie on t
building t takes 0.70240515s
building trie on mc
building mc takes 0.44976747s
building trie on k
building k takes 0.0000065s
building trie on cn
building cn takes 0.00758475s
join takes 1.129867916s
total takes 2.289817
output [[Str("& Teller")]]
running query 3a 
Query: 3a
Loading movie_keyword
Loading mi
Loading t
Loading k
building flat table on mi
building mi takes 0.000010416s
building trie on t
building t takes 0.2426155s
building trie on mk
building mk takes 0.40053034s
building trie on k
building k takes 0.000007208s
join takes 34.248333ms
total takes 0.67751044
output [[Str("2 Days in New York")]]
running query 3b 
Query: 3b
Loading movie_keyword
Loading k
Loading t
Loading mi
building flat table on mi
building mi takes 0.000006666s
building trie on t
building t takes 0.07970579s
building trie on mk
building mk takes 0.4260887s
building trie on k
building k takes 0.000006083s
join takes 465.791Âµs
total takes 0.50635314
output [[Str("300: Rise of an Empire")]]
running query 3c 
Query: 3c
Loading movie_keyword
Loading k
Loading t
Loading mi
building flat table on mi
building mi takes 0.000014625s
building trie on t
building t takes 0.48798332s
building trie on mk
building mk takes 0.3966372s
building trie on k
building k takes 0.000005916s
join takes 221.846375ms
total takes 1.1065849
output [[Str("& Teller 2")]]
running query 4a 
Query: 4a
Loading movie_keyword
Loading t
Loading mi_idx
Loading it
Loading k
building flat table on mk
building mk takes 0.000003958s
building trie on t
building t takes 0.29290685s
building trie on mi_idx
building mi_idx takes 0.12473075s
building trie on k
building k takes 0.000009708s
building trie on it
building it takes 0.000004083s
join takes 789.108583ms
total takes 1.2068927
output [[Str("& Teller 2")], [Str("5.1")]]
running query 4b 
Query: 4b
Loading movie_keyword
Loading k
Loading t
Loading mi_idx
Loading it
building flat table on mk
building mk takes 0.000005916s
building trie on t
building t takes 0.08040604s
building trie on mi_idx
building mi_idx takes 0.001772166s
building trie on k
building k takes 0.000005333s
building trie on it
building it takes 0.000002833s
join takes 708.537833ms
total takes 0.79079354
output [[Str("Batman: Arkham City")], [Str("9.1")]]
running query 4c 
Query: 4c
Loading movie_keyword
Loading mi_idx
Loading t
Loading it
Loading k
building flat table on mk
building mk takes 0.000005541s
building trie on t
building t takes 0.43093348s
building trie on mi_idx
building mi_idx takes 0.17330763s
building trie on k
building k takes 0.000009666s
building trie on it
building it takes 0.000004875s
join takes 948.693083ms
total takes 1.5530875
output [[Str("& Teller 2")], [Str("2.1")]]
running query 5c 
Query: 5c
Loading t
Loading mc
Loading ct
Loading mi
Loading info_type
building flat table on mi
building mi takes 0.000006833s
building trie on t
building t takes 0.5318221s
building trie on mc
building mc takes 0.03683833s
building trie on it
building it takes 0.00001875s
building trie on ct
building ct takes 0.000004833s
join takes 244.982666ms
total takes 0.8138129
output [[Str("11,830,420")]]
running query 11a 
Query: 11a
Loading movie_keyword
Loading k
Loading mc
Loading ct
Loading cn
Loading movie_link
Loading lt
Loading t
building flat table on mk
building mk takes 0.000028625s
building trie on t
building t takes 0.20397615s
building trie on ml
building ml takes 0.002236583s
building trie on mc
building mc takes 0.27675873s
building trie on lt
building lt takes 0.000011541s
building trie on k
building k takes 0.000003416s
building trie on ct
building ct takes 0.0000035s
building trie on cn
building cn takes 0.006987416s
join takes 850.765833ms
total takes 1.3409575
output [[Str("Batman Beyond")], [Str("followed by")], [Str("Churchill Films")]]
running query 11b 
Query: 11b
Loading movie_keyword
Loading k
Loading movie_link
Loading t
Loading mc
Loading cn
Loading lt
Loading ct
building flat table on mk
building mk takes 0.000005s
building trie on t
building t takes 0.000009583s
building trie on ml
building ml takes 0.001852666s
building trie on mc
building mc takes 0.26194385s
building trie on lt
building lt takes 0.000008416s
building trie on k
building k takes 0.000003375s
building trie on ct
building ct takes 0.000003166s
building trie on cn
building cn takes 0.006952916s
join takes 669.721416ms
total takes 0.94063634
output [[Str("The Money Man")], [Str("follows")], [Str("Filmlance International AB")]]
running query 11c 
Query: 11c
Loading movie_keyword
Loading k
Loading t
Loading movie_link
Loading link_type
Loading mc
Loading cn
Loading ct
building flat table on mk
building mk takes 0.000010708s
building trie on t
building t takes 0.7067509s
building trie on ml
building ml takes 0.002061208s
building trie on mc
building mc takes 0.376833s
building trie on lt
building lt takes 0.000006458s
building trie on k
building k takes 0.000002541s
building trie on ct
building ct takes 0.000002458s
building trie on cn
building cn takes 0.000018416s
join takes 919.510416ms
total takes 2.005343
output [[Str("24")], [Str("(1997-2002) (worldwide) (all media)")], [Str("20th Century Fox Home Entertainment")]]
running query 11d 
Query: 11d
Loading movie_keyword
Loading mc
Loading k
Loading cn
Loading movie_link
Loading ct
Loading t
Loading link_type
building flat table on mk
building mk takes 0.000005291s
building trie on t
building t takes 0.64940405s
building trie on ml
building ml takes 0.002069958s
building trie on mc
building mc takes 0.3794033s
building trie on lt
building lt takes 0.000022208s
building trie on k
building k takes 0.000003875s
building trie on ct
building ct takes 0.000003333s
building trie on cn
building cn takes 0.036417585s
join takes 910.055958ms
total takes 1.9779353
output [[Str("...denn sie wissen nicht, was sie tun")], [Str("(1954) (UK) (TV)")], [Str("13th Street")]]
running query 12a 
Query: 12a
Loading movie_companies
Loading cn
Loading t
Loading mi_idx
Loading mi
Loading it1
Loading it2
Loading ct
building flat table on mi
building mi takes 0.000004916s
building trie on t
building t takes 0.08745646s
building trie on mi_idx
building mi_idx takes 0.009820375s
building trie on mc
building mc takes 0.56440943s
building trie on it2
building it2 takes 0.000006125s
building trie on it1
building it1 takes 0.0000035s
building trie on ct
building ct takes 0.000003125s
building trie on cn
building cn takes 0.013940625s
join takes 58.089791ms
total takes 0.7338851
output [[Str("3:20")], [Str("8.1")], [Str("10th Grade Reunion Films")]]
running query 12b 
Query: 12b
Loading movie_info
