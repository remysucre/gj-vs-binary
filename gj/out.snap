output: Str("(A Warner Bros.-First National Picture) (presents)")Str("A Clockwork Orange")Num(1934)
output: Str("(Set Decoration Rentals) (uncredited)")Str("Disaster Movie")Num(2008)
output: Str("(co-production)")Str("Intouchables")Num(2011)
output: Str("(Set Decoration Rentals) (uncredited)")Str("Disaster Movie")Num(2004)
output: Str("'Doc'")
output: Str("'Doc'")
output: Str("& Teller")
output: Str("2 Days in New York")
output: Str("300: Rise of an Empire")
output: Str("& Teller 2")
output: Str("5.1")Str("& Teller 2")
output: Str("9.1")Str("Batman: Arkham City")
output: Str("2.1")Str("& Teller 2")
output: Str("11,830,420")
output: Str("Churchill Films")Str("followed by")Str("Batman Beyond")
output: Str("Filmlance International AB")Str("follows")Str("The Money Man")
output: Str("20th Century Fox Home Entertainment")Str("(1997-2002) (worldwide) (all media)")Str("24")
output: Str("13th Street")Str("(1954) (UK) (TV)")Str("...denn sie wissen nicht, was sie tun")
output: Str("10th Grade Reunion Films")Str("8.1")Str("3:20")
output: Str("$10,000")Str("Birdemic: Shock and Terror")
output: Str("\"Oh That Gus!\"")Str("7.1")Str("$1.11")
output: Str("Afghanistan:24 June 2012")Str("1.0")Str("&Me")
output: Str("501audio")Str("1.8")Str("5 Time Champion")
output: Str("DL Sites")Str("1.8")Str("Champion")
output: Str("\"O\" Films")Str("1.0")Str("#54 Meets #47")
output: Str("1.0")Str("$lowdown")
output: Str("6.4")Str("Of Dolls and Murder")
output: Str("1.0")Str("$lowdown")
output: Str("USA:1 June 2007")Str("Battlestar Galactica: The Resistance")
output: Str("USA:27 April 2007")Str("RoboCop vs Terminator")
output: Str("USA:1 April 2003")Str("24: Day Six - Debrief")
output: Str("(Not So) Instant Photo")Str("06/05")
output: Str("Det Danske Filminstitut")Str("followed by")Str("Der Serienkiller - Klinge des Todes")
output: Str("Filmlance International AB")Str("followed by")Str("HÃ¤mndens pris")
output: Str("Churchill Films")Str("followed by")Str("Batman Beyond")
output: Str("alternate language version of")Str("12 oz. Mouse")Str("'Angel': Season 2 Overview")
